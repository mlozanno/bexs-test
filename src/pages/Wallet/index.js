import React from 'react';

const Wallet = () => <div data-testid="wallet-page" />;

export default Wallet;
