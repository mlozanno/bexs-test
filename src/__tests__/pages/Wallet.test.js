import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import Wallet from '~/pages/Wallet';

describe(`<Wallet /> page`, () => {
	it(`should be a function`, () => {
		expect(typeof Wallet).toBe(`function`);
	});

	it(`should be render component`, () => {
		const { getByTestId } = render(<Wallet />);

		const wallet = getByTestId(`wallet-page`);

		expect(wallet).toMatchSnapshot();
	});
});
