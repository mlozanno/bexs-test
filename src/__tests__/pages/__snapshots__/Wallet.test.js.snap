// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wallet /> page should be render component 1`] = `
<div
  data-testid="wallet-page"
/>
`;
